mmv1/products/bigquerydatapolicy/terraform.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  DataPolicy: !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: projects/{{project}}/locations/{{location}}/dataPolicies/{{data_policy_id}}
+    import_format:
+      [
+        "projects/{{project}}/locations/{{location}}/dataPolicies/{{data_policy_id}}",
+        "{{data_policy_id}}",
+      ]
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "bigquery_datapolicy_data_policy_basic"
+        min_version: beta
+        primary_resource_id: "data_policy"
+        primary_resource_name: 'fmt.Sprintf("tf_test_data_policy%s", context["random_suffix"])'
+        vars:
+          data_policy_id: "data_policy"
+          taxonomy: "taxonomy"
+    properties:
+      policyTag: !ruby/object:Overrides::Terraform::PropertyOverride
+        diff_suppress_func: "projectNumberDiffSuppress"
```

Comment:
Is this one where the API accepts the project_id but then returns the number?

Reply:
yes